app/konnect/regional-consumers.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Regional Consumers
```

Comment:
Our official name for this initiative is "Consumers". What is the best way for us to reflect that this initiative is an expansion of the existing consumer entity such that consumers can be stored in a centralized space per region?

Reply:
In my revision, I'll call them "centralized consumer management" or "centrally managed consumers" if I need to call out the difference between them and "original" consumers. Otherwise, I'll use "consumers" when the context makes it clear we're talking about the central ones.